app/views/chess_boards/show.html.erb

```diff
@@ -12,37 +12,72 @@
   </head>
   <body>
     <br><br><br>
-
+    
     <div class="chessboard">
 
         <% (0..7).each do |row| %>
           <div id="row">
-         <% (0..7).each do |column| %>
+        <% (0..7).each do |column| %>
            <% if (row % 2 == 0) %>
              <% if (column % 2 == 0) %>
-                <div class="white"></div>
-             <% else %>
-                <div class="black"></div>
-             <% end %>
-          <% else %>
-             <% if (column % 2 == 1) %>
-                <div class="white"></div>
-             <% else %>
-                <div class="black"></div>
-             <% end %>
-           <% end %>
-         <% end %>
-          </div>
-        <% end %>
 
-    </div>
+                <div class="white">
+                <% @pieces.each do |piece| %>
+                  <% if piece.x_position == column && piece.y_position == row %>
+                    <div class="piece" display-piece-id="<%piece.id%>">
```

Comment:
Got it working (with Matt's help). All is good. Will write tests for each piece to reflect their attributes in their respective models. Need to update my branch with latest master to get Joel's fixes on deleting chessboard#controller, chessboard#show.